.gcloudignore

```diff
@@ -16,4 +16,7 @@
 # Python pycache:
 __pycache__/
 # Ignored by the build system
-/setup.cfg
\ No newline at end of file
+/setup.cfg
+.media
+.env
+.vscode
```

Comment:
There are a bunch of conflicts in the files. Apologies if you already know this stuff. You can handle the conflicts in github or in vscode. It's a little easier to do in vscode and you can resolve them by doing a pull to master then merging master into your branch.